client/web/src/repo/settings/RepoSettingsMirrorPage.tsx

```diff
@@ -207,6 +210,70 @@ const CheckMirrorRepositoryConnectionActionContainer: React.FunctionComponent<
     )
 }
 
+// Add interface for props then create component
+interface CorruptionLogProps {
+    repo: SettingsAreaRepositoryFields
+    history: H.History
+}
+
+const CorruptionLogsContainer: React.FunctionComponent<CorruptionLogProps> = props => {
+    const health = props.repo.mirrorInfo.isCorrupted ? (
+        <Alert className={classNames('mb-0', styles.alert)} variant="danger">
+            The repository is corrupt, check the log entries below for more info and consider recloning.
+        </Alert>
+    ) : (
+        <Alert className={classNames('mb-0', styles.alert)} variant="success">
+            The repository is currently not corrupt
```

Comment:
nit (the ending period is not a nit :)

```suggestion
            The repository is currently in healthy state.
```

Reply:
I've removed this as part of feedback from @rrhyne 😄